tests/testthat/test-zzzz-bs-sass.R

```diff
@@ -1,8 +1,12 @@
+local_disable_cache()
+
 test_that("Can access the sass behind all versions and Bootswatch themes", {
+  skip_on_cran()
```

Comment:
What's the rationale for this?

Reply:
These tests take forever and by the time CRAN gets them, it would be too late. At least we can do these tests on GHA